civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5799,6 +5799,16 @@ function civihr_employee_portal_user_login_submit(&$form, &$form_state) {
     }
 }
 
+/**
+ * Implements hook_user_login
+ */
+function civihr_employee_portal_user_login(&$edit, $account) {
+  global $base_url;
+  if (!isset($_POST['form_id']) || $_POST['form_id'] != 'user_pass_reset') {
+    $_GET['destination'] = user_access('access CiviCRM') ?  'civicrm/tasksassignments/dashboard#/tasks' : 'dashboard';
```

Comment:
Is this really the best way to do this? Usually, it's a bad practice to change the values from global variables such as $_GET, $_POST, $_SERVER etc.

Have you checked the [user_login_destination](https://api.drupal.org/api/drupal/modules!user!user.module/function/user_login_destination/7.x) function? It looks like it does what you're trying to achieve here.